db/migrate/20171114183347_add_lock_flags_to_maps.rb

```diff
@@ -0,0 +1,10 @@
+Sequel.migration do
+
+  change do
```

Comment:
Write explicit `def up` and `def down` methods, or this behavior will be subject to whatever the next sequel gem update brings. Explicit is always best :)

Reply:
I did make sure to test this beforehand with `db:migrate:up` and `db:migrate:down`.  If you think Sequel would introduce breaking changes to this functionality, I can change it.

Reply:
Explicit is always safer